webapp/src/components/properties/lastModifiedAt/lastModifiedAt.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react'
+
+import {Card} from '../../../blocks/card'
+import {CardTree} from '../../../viewModel/cardTree'
+
+const moment = require('moment')
+
+type Props = {
+    card: Card,
+    cardTree: CardTree | undefined
+}
+
+const LastModifiedAt = (props: Props): JSX.Element => {
+    let latestBlock = props.cardTree ? props.cardTree.allBlocks.filter((block) => block.parentId === props.card.id || block.id === props.card.id).sort((a, b) => b.updateAt - a.updateAt)[0] : props.cardTree
+    latestBlock = latestBlock || props.card
```

Comment:
I'm gonna say lets opt to use if statement. Although more lines of code, I feel the readability will increase.
Alternative is
```suggestion
    let latestBlock = props.card
    if (props.cardTree) {
        latestBlock = props.cardTree.allBlocks
        .filter((block) => block.parentId === props.card.id || block.id === props.card.id)
        .sort((a, b) => b.updateAt - a.updateAt)[0]
   }
```

I was trying to move each function call to a new line for better readability like the .filter and .sort in my revision above.

Reply:
I really like how it looks. Done.